structural-annotation.wdl

```diff
@@ -21,16 +21,20 @@ workflow s_annotate {
   String  database_location
   String  container
   String? gm_license
+  String pre_qc_container = "microbiomedata/mg-annotation:0.1.2"
+
+
 
   if(pre_qc_execute) {
     call pre_qc {
       input:
         project_type = imgap_project_type,
         input_fasta = imgap_input_fasta,
         project_id = imgap_project_id,
-        container=container
+        container=pre_qc_container
```

Comment:
same question here about containers

Reply:
pre-qc container was what was listed at some point, i'll test both instances with the main container and remove if no issues pop up